.travis.yml

```diff
@@ -2,21 +2,20 @@ language: node_js
 node_js:
     - "0.12"
     - "0.10"
-#   - "0.8"   # no longer supported by iconv
-#   - "0.6"   # no longer supported by async
-    - "iojs-v1.8.1"
-    - "iojs"
-
+    - "stable"
+    - "iojs-v1"
+    - "iojs-v2"
+    - "iojs-v3"
```

Comment:
ok but iojs gets future updates... i dont know if only for bugs or also for new features. So I think that iojs-v3 should remain there.